### PR TITLE
Merge template updates up to 2.14.1

### PR DESCRIPTION
**Pull request by christopher-mohr**
* Merging the last two template updates
* Add check if protein information available from BioMart

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/epitopeprediction/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/epitopeprediction _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by christopher-mohr** [comment]
Ensembl most recent BioMart version currently unavailable, therefore the `grch38` test fails:

``` 
Server Status
The Ensembl websites and REST servers are currently unavailable as we apply updates. Our archive sites, FTP site and public MySQL servers are unaffected. Please try again shortly.
```

**Review by jonasscheid — CHANGES_REQUESTED** [review]
LGTM, thanks! Only adjusting the changelog date and name remains

**Comment by christopher-mohr** [comment]
> LGTM, thanks! Only adjusting the changelog date and name remains

I will still check the issue with GRCH38 which does not seem to be (solely) caused by the Ensembl unavailability.

**Comment by christopher-mohr** [comment]
In the meantime, Ensembl replied and confirmed that there are currently some issues with the most recent BioMart version. What happened is that the protein field/table we are using to retrieve protein information was/is currently not accessible and therefore no information is available in the df. 

I added a check to make sure that the information is only accessed if available.

File: bin/epaa.py [hunk 1]
```diff
@@ -425,6 +425,11 @@ def read_lig_ID_values(filename):
 
 def create_protein_column_value(pep):
     # retrieve Ensembl protein ID for given transcript IDs, if we want to provide additional protein ID types, adapt here
+
+    # we have to catch cases where no protein information is available, e.g. if there are issues on BioMart side
+    if transcriptProteinTable is None:
+        return ""
```
[the remark this answers is not in the document]
Comment (christopher-mohr) [reply bin/epaa.py:431]: added

**Pull request merged**